src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,6 +33,22 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        
+        //Split user-entered address
+        address.replace("/a","");
+        splittedAddress = address.split(", ");
```

Comment:
Rather than above code, consider split by comma then trim the space which is more user-friendly. Since user might often forget to put a space after comma.